apps/rush/src/actions/ChangeAction.ts

```diff
@@ -334,7 +333,7 @@ export default class ChangeAction extends BaseRushAction {
   private _writeChangeFile(changeFile: IChangeFile): Promise<void> {
     const output: string = JSON.stringify(changeFile, undefined, 2);
 
-    const filePath: string = this._generateChangeFilePath(changeFile.packageName);
+    const filePath: string = _generateChangeFilePath(this.rushConfiguration, changeFile.packageName);
```

Comment:
>_generateChangeFilePath [](start = 29, length = 23)

I'm confused. Shouldn't `rush` be using `_writeChangeFileSync()` and `_generateChangeFilePath` should be a private/un-exported function?